### PR TITLE
WIP windows: build tests, run some tests

### PR DESCRIPTION
**Pull request by Anton-4**

### CODE REVIEW

**Comment by Anton-4** [comment]
Let's keep this open

File: .github/workflows/windows.yml [hunk 1]
```diff
@@ -15,10 +15,16 @@ jobs:
     runs-on: windows-2022
     env:
       LLVM_SYS_130_PREFIX: C:\LLVM-13.0.1-win64
-    timeout-minutes: 90
+    timeout-minutes: 150
```
Comment (folkertdev) [inline .github/workflows/windows.yml:18]: hmm windows is slow huh

**Review by folkertdev — APPROVED** [review]
we should now be able to run more tests (gen and cli_run benchmarks), but this is a good start!

**Comment by Anton-4** [comment]
Yeah I included `cli_run` before but I was getting a warning that came up in stderr and failed the test:
```
warning: ignoring debug info with an invalid version (0) in app\r\n
```

**Pull request merged**